angr/engines/concrete.py

```diff
@@ -0,0 +1,152 @@
+import logging
+import signal
+import sys
+from angr.engines import SimEngine
+from angr_targets.concrete import ConcreteTarget
+
+
+l = logging.getLogger("angr.engines.concrete")
```

Comment:
You want to use `_l` for file-level loggers so the linter won't complain. Sorry that this wasn't clearly communicated.

Reply:
since when does the linter complain about variables named `l`...

Reply:
A not-so-recent update of pylint or PyCharm started to complain about single-letter variables being used as a global variable.

Reply:
We've disabled that in pylint, and I disabled it in my pycharm, and that's the way it should be!